Tilers/__init__.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+
+# Note: order matters and must respect the dependency (e.g. inheritance) tree
+from .kd_tree import kd_tree
+from .object_to_tile import *
+
+
```

Comment:
We miss some tests !
Refer to [this issue](https://github.com/VCityTeam/py3dtiles-tilers/issues/6)